src/Components/NavBar.js

```diff
@@ -0,0 +1,13 @@
+import "./NavBar.css";
+
+const NavBar = () => {
+
+    return (<ul id="nav">
```

Comment:
Great work! small style note: follow the same convention as your other return statements by adding a new line after the opening paren and before the closing paren